### ISSUE TITLE
[Accessibility][Keyboard Navigation-Flow Chart]: Keyboard focus gets trapped on the flow chart.

### ISSUE THREAD

**Issue by ashish315** [microsoft/BotFramework-Composer#2021]
**User Experience** 
If the focus order is not correct i.e. if focus will get stuck at a particular button & will not move forward then it can confuse and disorient the user hindering his ability to complete his task.

**Test Environment**
Browser: Microsoft Edge Canary Version: 82.0.425.0 (Official build)
OS build: 2004 (19564.1005)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Repro Steps:** 
1. Open the above mentioned URL in EDGE.
2. Select 'Design Flow' from left navigation.
3. Now, try to navigate with tab till the flow chart.
4. Observe the issue when u try to access the flow chart.

**Actual:** 
As soon as focus reaches to the design flow chart, user gets trapped. Keyboard focus does not move forward nor in backward direction. The keyboard focus gets trapped on the design flow chart.

**Expected:**
The expected behavior is the on pressing tab the focus should moves to the next interactive.
Ex- Design flow chart-->'+' button (Then navigation within the chart should be by up/down arrow)-->next tab property pane at right.

**Note:**
1. Due to the trap the user will not be able to navigate within the flow chart, in order to navigate inside the flow chart user will have to use the mouse.
2. Same Issue observed at the end node, in scan mode the focus gets stuck when user navigate in backward navigation.

**MAS Impacted:** [MAS2.1.2](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={ebdc3fe1-09a9-4aa2-a29b-e97764774e6c})

**Attachment:**
[MAS2.1.2_keyboard trap at design flow chart.zip](https://github.com/microsoft/BotFramework-Composer/files/4229139/MAS2.1.2_keyboard.trap.at.design.flow.chart.zip)

**Comment by ashish315** [comment]
As checked the issue is resolved so HCL is closing the issue.
